core/app/services/uffizzi_core/logs_service.rb

```diff
@@ -22,7 +23,7 @@ def request_logs(container, query)
       controller_client.deployment_container_logs(
         deployment_id: deployment.id,
         container_name: UffizziCore::ContainerService.pod_name(container),
-        limit: query[:limit],
+        limit: query[:limit] || Settings.container.default_logs_limit,
```

Comment:
Does this `||` logical OR work as expected if no parameter is required?

Reply:
Yes, it returns the default value if query[:limit] is nil